docs/headless-cms/08-01-add-clusterdocstopic.md

```diff
@@ -0,0 +1,186 @@
+- Documents start with metadata, including obligatory `Title` for the document name and optional `Type` for any additional document grouping.
+- Documents are written in Markdown and follow a simple Markdown syntax. Any customizations can result in incorrect images or links rendering. To avoid such issues, use our [content guidelines](https://github.com/kyma-project/community/tree/master/guidelines/content-guidelines).
+
+This tutorial shows how you can customize the `docs-ui` view that is available in the Console UI under the ![](./assets/docs.png) icon on the top navigation panel. The purpose of this tutorial is to create a new **Prometheus** documentation section that contains **Concepts** and **Guides** topic groups and a set of Markdown subdocuments. The document sources used in the ClusterDocsTopics point to specific topics in the official Prometheus documentation.  
+
+>**NOTE:** When you apply multiple ClusterDocsTopics, sort them using the **order** label. Otherwise, the Console UI displays the topics in the order in which you apply them.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Steps
+
+1. Install Kyma.
+
+2. In the terminal, apply these ClusterDocsTopic custom resources:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: docs-ui
+    cms.kyma-project.io/group-name: prometheus
+    cms.kyma-project.io/order: "1"
+  name: prometheus-concepts
+spec:
+  displayName: "Concepts"
+  description: "Some docs about Prometheus concepts"
+  sources:
+    - type: markdown
+      name: docs
+      mode: package
+      url: https://github.com/prometheus/docs/archive/master.zip
+      filter: content/docs/concepts
+---
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: docs-ui
+    cms.kyma-project.io/group-name: prometheus
+    cms.kyma-project.io/order: "2"
+  name: prometheus-guides
+spec:
+  displayName: "Guides"
+  description: "Some docs about Prometheus guides"
+  sources:
+    - type: markdown
+      name: docs
+      mode: package
+      url: https://github.com/prometheus/docs/archive/master.zip
+      filter: content/docs/guides
+EOF
+```
+
+3. Check the status of custom resources:
+
+```
+kubectl get clusterdocstopics
+```
+
+The custom resources should be in the `Ready` phase:
+
+```
+NAME                                   PHASE   AGE
+prometheus-concepts                    Ready   59s
+prometheus-guides                      Ready   59s
+```
+
+If a given custom resource is in the `Ready` phase and you want to get details of the created ClusterAssets, such as document names and the location of Minio buckets, run this command:
+
+```
+kubectl get clusterasset -o yaml -l cms.kyma-project.io/docs-topic=prometheus-concepts
+```
+
+The result is as follows:
```

Comment:
please describe the result, important to notice is that the information about documentation is in status  of the resource

so if CluserAsset phase is ready Ready, then you can find information like `baseUrl`and then all the files with the metadata info, like:

```
files:
      - metadata:
          sort_rank: 1
          title: Data model
        name: docs-master/content/docs/concepts/data_model.md
```

this is very crucial info